### ISSUE TITLE
Company backend issue

### ISSUE THREAD

**Issue by drot** [holomorph/systemd-mode#4]
When enabling `company-mode` we get the following error while `systemd-mode` is active:

```
Company backend ’t’ could not be initialized:
Symbol’s function definition is void: t
```
`GNU Emacs 25.1.1 (x86_64-unknown-linux-gnu, X toolkit, Xaw scroll bars) of 2016-08-22`

**Comment by holomorph** [comment]
Should be fixed by a73c16a

**Comment by drot** [comment]
Fixed, thanks.
